osbs/build/spec.py

```diff
@@ -315,31 +315,33 @@ def set_params(self, git_uri=None, git_ref=None,
         self.name.value = make_name_from_git(self.git_uri.value, self.git_branch.value)
         self.group_manifests.value = group_manifests or False
         self.prefer_schema1_digest.value = prefer_schema1_digest
-        if not base_image:
-            raise OsbsValidationException("base_image must be provided")
-        self.trigger_imagestreamtag.value = get_imagestreamtag_from_image(base_image)
-        self.builder_build_json_dir.value = builder_build_json_dir
```

Comment:
This one doesn't belong here - and breaks worker builds for flatpaks :( Fixed in #692